tmt/utils.py

```diff
@@ -2861,8 +2861,18 @@ def public_git_url(url: str) -> str:
     :returns: URL that is publicly accessible without authentication,
         or the original URL if no applicable conversion was found.
     """
+    return modify_git_url(url, PUBLIC_GIT_URL_PATTERNS)
 
-    for pattern, replacement in PUBLIC_GIT_URL_PATTERNS:
+
+def modify_git_url(url: str, patterns: List[Tuple[str, str]]) -> str:
+    """
+    Modify git url based on supplied patterns
+
+    :param url: an URL to modify
+    :param patterns: List of pattern to try in order
```

Comment:
"patternS"

Reply:
fixed in [de05583](https://github.com/teemtee/tmt/pull/2223/commits/de055837d6454899dacf47e2a499926e630ab043)